### PR TITLE
Move to readthedocs as main homepage

### PR DESCRIPTION
**Pull request by hoyer-a**
Added general description, cards for packages, contributing and supporters section.

### CODE REVIEW

**Review by f-brinkmann — CHANGES_REQUESTED** [review]
Thanks for implementing - I looks quite nice. The only general thing is that we should find a way to control the size of the images in the *packages* and *supporters* sections. Is there a way to do add image margins in the code? Otherwise, we could make logos that have a specific size for each section.

**Comment by hoyer-a** [comment]
Thanks for the feedback. 

Image size can be changed now, but there seems to be no standard option for margins.

File: docs/_templates/navbar-nav.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                        </li>
+
+                        <li class="nav-item">
+                          <a class="nav-link nav-internal" href="examples_gallery.html">
+                            Examples gallery
+                          </a>
+                        </li>
+
+                        <li class="nav-item">
+                          <a class="nav-link nav-external" href="https://sofar.readthedocs.io/en/stable/">
```
Comment (ahms5) [inline docs/_templates/navbar-nav.html:18]: maybe put pyfar before sofar.

File: docs/conf.py [hunk 2]
```diff
@@ -55,6 +58,33 @@
...
+html_title = "pyfar"
+html_favicon = '_static/favicon.ico'
+
+
+# -- HTML theme options
+# https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/layout.html
+
+html_theme_options = {
+    "navbar_start": ["navbar-logo"],
+    "navbar_end": ["theme-switcher", "navbar-icon-links"],
```
Comment (ahms5) [inline docs/conf.py:72]: Is it possible to fix the theme to light? Until we don't have extra figures for dark.

File: docs/index.rst [hunk 3]
```diff
@@ -3,23 +3,193 @@
...
-Pyfar gallery
+   .. grid-item-card::  
+      :text-align: center
+      :shadow: lg
+
+      **coming next...**
+      ^^^^
+      The pyfar base package will be extendend an new packages for audio 
+      input/output to external hardware, acoustic measurements, 
+      and spherical array processing are planned.
```
Comment (ahms5) [inline docs/index.rst:167]:
```suggestion
      and material modeling are planned.
```
spharpy is already there, so we can point towards imkar

**Review by ahms5 — CHANGES_REQUESTED** [review]
Thank you very much, it looks really nice. I have just some small comments.

An other thing is that the circleci does not seem to work, readthedocs is working. 
Does anybody have an idea why? (maybe because the branch is on your repo)

File: docs/conf.py (hunk 2 above)
Comment (hoyer-a) [reply docs/conf.py]: Yep, it's fixed to light now.

File: docs/index.rst [hunk 4]
```diff
@@ -3,38 +3,280 @@
...
-Pyfar gallery
+               code
+
+   .. grid-item-card::  
+      :text-align: center
+      :shadow: lg
+
+      **coming next...**
+      ^^^^
+      The pyfar base package will be extendend an new packages for audio
```
Comment (sikersten) [inline docs/index.rst:165]:
```suggestion
      The pyfar base package will be extendend. New packages for audio 
```

File: docs/index.rst [hunk 5]
```diff
@@ -3,38 +3,280 @@
...
+
+   .. grid-item-card:: Python Room Acoustics Tools
+      :text-align: center
+      :shadow: lg
+
+      .. image:: resources/logos/pyfar_logos_fixed_size_pyrato.png
+         :width: 300 px
+
+      ^^^^
+      A collection of functions for commonly used operations in room acoustics
```
Comment (sikersten) [inline docs/index.rst:137]:
```suggestion
      A collection of functions for commonly used operations in room acoustics.
```

File: docs/index.rst [hunk 6]
```diff
@@ -3,38 +3,280 @@
...
+
+   .. grid-item-card::  Spherical array processing in python
+      :text-align: center
+      :shadow: lg
+   
+      .. image:: resources/logos/pyfar_logos_fixed_size_spharpy.png
+         :width: 300 px
+
+      ^^^
+      Spherical array processing in python
```
Comment (sikersten) [inline docs/index.rst:107]: @mberz, @ahms5 Replace with describing sentence?

File: docs/index.rst [hunk 7]
```diff
@@ -3,23 +3,193 @@
...
+            .. button-link:: https://github.com/pyfar
+               :color: primary
+               :expand:
+               :shadow:
+ 
+               progress 
+
+
+|
+Pyfar gallery
```
[the remark this answers is not in the document]
Comment (sikersten) [reply docs/index.rst:181]: +1

**Review by sikersten — COMMENTED** [review]
Looks great, thank you very much! I have only minor comments

**Review by pingelit — COMMENTED** [review]
Thank you for implementing this! It looks very nice.

I only found the things that sikersten found.

File: docs/index.rst (hunk 6 above)
Comment (sikersten) [reply docs/index.rst:107]: For now, maybe remove the sentence in normal font and just keep the bold one?

**Review by sikersten — APPROVED** [review]

**Pull request merged**